### DISCUSSION TITLE
Remote connection

### DISCUSSION THREAD

**Discussion by Goddard** [opcodesio/log-viewer#120] [Q&A]
I'd like to use this with an API.  It has no "web" routes.  Would be nice if you could connect to a remote server and just run this locally and manage multiple servers from one interface.

**Comment by 99kennetn** [comment]
This would be awsome :) 
i just found this package so still learning about it so i dont know if it already has it, but it seems to be the same problem as this wish. 
A project that is run on multiple servers, could be nice to have a collected view of the logs from all the projects :D

**Comment by rbatinov** [comment]
After version 2, there is such an option, documented [here](https://log-viewer.opcodes.io/docs/2.x/configuration/multiple-hosts) .
